pre-commit/app_tests/json_tests.py

```diff
@@ -27,23 +27,58 @@ def __init__(self, app_repo_name, repo_location, **kwargs):
 
         self._app_is_certified = self._parser.app_json["publisher"] == "Splunk"
 
+    @classmethod
+    def format_as_index(indices: list[Any], container: str = "schema") -> str:
```

Comment:
First arg to a classmethod is gonna be the class. Let's make it static
```suggestion
    @staticmethod
    def format_as_index(indices: list[Any], container: str = "schema") -> str:
```

Reply:
Makes sense changing